pkgs/games/trackballs/default.nix

```diff
@@ -1,24 +1,23 @@
 { stdenv, fetchFromGitHub, cmake, SDL2, SDL2_ttf, gettext, zlib, SDL2_mixer, SDL2_image, guile, libGLU, libGL }:
 
-with stdenv.lib;
-
 stdenv.mkDerivation rec {
   pname = "trackballs";
   version = "1.3.1";
 
   src = fetchFromGitHub {
     owner = "trackballs";
-    repo = "trackballs";
+    repo = pname;
```

Comment:
I am not against or for this style but when usually the diff is not worth it. Just a little remark for the future.

Reply:
Ok, I will keep that in mind.